Blockzilla.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -39,7 +39,7 @@
       },
       {
         "package": "SnapKit",
-        "repositoryURL": "https://github.com/SnapKit/SnapKit.git",
```

Comment:
nit: not sure this change is required but should have no impact in general

Reply:
Yeah I think Xcode did this on its own when regenerating the `Package.resolve` since the URL in the project file didn't have the `.git` 😬